pages/index.tsx

```diff
@@ -22,10 +66,10 @@ const Home: NextPage = () => {
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <main className={styles.main}>
-        <h1 className={styles.title}>
+      <Main>
+        <Title className={styles.title}>
           Welcome to <a href="/about">Prolog</a>
-        </h1>
+        </Title>
 
         <p className={styles.description}>
```

Comment:
Why didn't you migrate this to styled-components?

Reply:
Come on this really boring stuff.

Reply:
Ok, lets go on